### ISSUE TITLE
Why TimeDistributed right after Embedding Layer?

### ISSUE THREAD

**Issue by pengpaiSH** [bradleypallen/keras-quora-question-pairs#2]
@bradleypallen Thanks for sharing your solution. In your network architecture, you add an TimeDistributed (Dense) after the word embedding layer. Could you please explain your motivation and why? In most cases, it seems that TimeDistributed is applied after a LSTM and before the Softmax?

**Comment by bradleypallen** [comment]
This is motivated by Stephen Merity's work cited in the README; the idea (as articulated by Stephen in a note I can't quite put my hands on at the moment) is to aid the transformation from word embedding space to sentence embedding space by putting a trainable layer on top of the output of an Embedding layer before the word embeddings get summed into a sentence embedding in the Lambda layer. I will admit that is arguably not a very good reason why in and of itself; removing the layer itself decreases model test accuracy by 1-2 points, so there's that.
